src/azure-cli/setup.py

```diff
@@ -142,7 +142,7 @@
     'PyGithub~=1.38',
     'PyNaCl~=1.5.0',
     'scp~=0.13.2',
-    'semver==2.13.0',
+    'semver~=2.13.0',
```

Comment:
Are you sure you need this pin? The last `2.x` release is almost five years old.

https://pypi.org/project/semver/2.13.0/

Maybe bump to `~=3.0.4`?

Reply:
Opened a separate ticket for this.

https://github.com/Azure/azure-cli/issues/31519